package.json

```diff
@@ -1,90 +1,35 @@
 {
     "name": "whispir-pairing-env",
-    "version": "0.1.0",
-    "description": "A starting point for Node.js express apps with TypeScript",
+    "version": "1.0.1",
+    "description": "A starter package for interview testing",
     "repository": {
         "type": "git",
-        "url": "https://github.com/Microsoft/TypeScript-Node-Starter"
+        "url": "https://www.whispir.com"
     },
-    "author": "Bowden Kelly",
-    "license": "MIT",
+    "author": "Whispir",
+    "license": "UNLICENSE",
     "scripts": {
-        "build": "npm run build-ts && npm run lint",
-        "build-ts": "tsc",
-        "debug": "npm run build && npm run watch-debug",
-        "lint": "tsc --noEmit && eslint \"**/*.{js,ts}\" --quiet --fix",
-        "serve-debug": "nodemon --inspect dist/server.js",
-        "serve": "node dist/server.js",
-        "start": "npm run serve",
         "test": "jest --forceExit --coverage --verbose",
-        "watch-ts": "tsc -w",
-        "watch": "concurrently -k -p \"[{name}]\" -n \"Sass,TypeScript,Node\" -c \"yellow.bold,cyan.bold,green.bold\" \"npm run watch-sass\" \"npm run watch-ts\" \"npm run watch-node\""
+        "test:watch": "jest --watch",
+        "build": "npm run build-ts",
+        "build-ts": "tsc",
+        "dev": "ts-node-dev --respawn src/server.ts",
+        "start": "node dist/server.js"
     },
     "dependencies": {
-        "@types/mongodb": "3.6.5",
```

Comment:
Heaps of unused/unnecessary packages. Better leave it out, let candidate add their own likes.